db/schema.rb

```diff
@@ -10,11 +10,39 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_07_16_194632) do
+ActiveRecord::Schema.define(version: 2020_07_27_214244) do
+
+  create_table "calendar_days", force: :cascade do |t|
+    t.string "calendar_id"
+    t.string "calendar_theme"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+  end
+
+  create_table "calendars", force: :cascade do |t|
+    t.string "rails"
+    t.string "generate"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.string "name"
+  end
+
+  create_table "memberships", force: :cascade do |t|
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+  end
+
+  create_table "song_entries", force: :cascade do |t|
```

Comment:
How will you know which day this record belongs to? I think you are missing a column on this model